modules/openapi-generator/src/main/resources/Javascript-Flowtyped/modelGeneric.mustache

```diff
@@ -12,7 +12,7 @@
  */
 export type {{classname}} = {
 {{#additionalPropertiesType}}
-    [key: string]: {{{additionalPropertiesType}}}{{#hasVars}} | any{{/hasVars}};
+    [key: string]: {{{additionalPropertiesType}}}{{#hasVars}} | unknown{{/hasVars}};
```

Comment:
I wonder if its a good idea to use `unknown` instead of `any`, since this probably breaks a lot of client code that accesses these values

Reply:
From the typescript perspective (ECMA standarts etc) it is better to use unknown, as it is exactly describe its state/type, which is unknown and you must check/know the type of it. IMO it make the code more error-free/robust in runtime as you will be forced to check the type.

Reply:
I can put it back to any but my linter and strict tsconfig settings do not like it, which is sign of not best typing.

Reply:
Or maybe, make use of config switch `supportsES6` that when it would be true, it would use `unknown` instead of `any`

Reply:
i would suggest to switch it back to `any`, in order not to break existing use cases.

Reply:
Consider using `unknown` instead of `any` here. The `unknown` type is safer because it requires us to perform explicit type checking before performing any operations on the value. This approach enhances type safety by ensuring we explicitly handle the variable's type before using it, thereby reducing the risk of runtime errors. Using `unknown` also improves code readability, as it clearly indicates that the type needs to be determined and handled carefully. This is especially beneficial in maintaining strict type safety practices across our TypeScript codebase.

Reply:
to avoid a breaking change, i.e. to preserve backwards compatibility, I would argue we should keep using `any` and not change it to `unknown`

Reply:
I also think preserving backwards compatibility is really important. @mvarchdev , could you make the change so we can merge the fix?

Reply:
Sorry for late reply, had a lot in work... I do not agree with that completely, preserving backwards compatibility si for sure important, but in our codebase, `any` type will fail in every project as we use strict typing config from official ts configs package. I suggest to add config switch to generator that we will be able to set if `any` of `unknown`, as default there would be `any` so preserving backwards compatibility, what do you think? @macjohnny @andreseloysv

Reply:
applying linting/strict typing rules etc. to generated code is a hard one anyway. you can always just have a custom script to post-process the output and replace `any` with `unknown`

a config switch sounds a bit like an overkill here, no?

Reply:
@macjohnny  We use that, because we have strict typing rule in our org :/ and tbh imo, having script to post-process the output and replace any with unknown is overkill as you would change already generated code, instead, with switch, you would just already set what will be generated in the end. In @sunray-eu We would really love to have this switch available. Let me know your final decision if it would be to big change in code with this switch adding, and then, we can finally merge it. Thanks!

Reply:
We have alternative package, of course, but 1. there is also build, 2. when we use it in other package, the return data type is sometimes any, which in lot of cases disable type check